core/src/main/java/org/hisp/dhis/android/core/analytics/trackerlinelist/internal/TrackerLineListParams.kt

```diff
@@ -37,6 +37,7 @@ internal data class TrackerLineListParams(
     val outputType: TrackerLineListOutputType?,
     val programId: String?,
     val programStageId: String?,
+    val trackedEntityType: String?,
```

Comment:
`trackedEntityTypeId`? We have `trackerVisualization` though, which makes it inconsistent anyway... I am not sure, up to you